openupgrade_scripts/scripts/base_automation/17.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 Viindoo Technology Joint Stock Company (Viindoo)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade
+
+
+def _ir_act_server_update_base_automation_id(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        ALTER TABLE ir_act_server
+            ADD COLUMN IF NOT EXISTS base_automation_id INTEGER;
+        """,
+    )
+    openupgrade.logged_query(
```

Comment:
I think we should make us of m2o_to_x2m() openupgradelib method

Reply:
I agree.